netlify.toml

```diff
@@ -3,5 +3,6 @@
 
 [[redirects]]
   from = "/webhook-handler"
-  to = "/.netlify/functions/handler"
+  # Set path to either javascript or go
+  to = "/.netlify/functions/javascript/"
```

Comment:
I was looking into whether this can be dynamic via `netlify`'s env variable, but that's not supported ([link](https://answers.netlify.com/t/using-environment-variables-in-redirect-query/16077)) -- likely due to constraints with `toml`